demo/shaders/glsl/pbr_frag.glsl

```diff
@@ -0,0 +1,968 @@
+layout (location = 0) in vec3 in_position_vs;
```

Comment:
I think it would be clearer if we keep all the shaders related to rendering the mesh feature prefixed with "mesh_"

I think this works well with most of the filenames (mesh_basic, mesh_wireframe). mesh_pbr would make more sense to me.

I also think we should avoid "basic" because it's very generic. Are there any changes other than not textured? It's still pulling in pbr. So maybe "mesh_pbr_untextured" and "mesh_pbr" would be accurate but succinct?

Reply:
Done.